doc/articles/toc.yml

```diff
@@ -429,35 +429,7 @@
     - name: Hot Reload
       href: xref:Uno.Platform.Studio.HotReload.Overview
     - name: Hot Design
-      href: xref:Uno.HotDesign.Overview
-      items:
```

Comment:
@carldebilly I think this "items:" is now missing here https://github.com/unoplatform/hd-docs/blob/36e76803a9334999827795f4e401675e9e333122/toc.yml#L1 as clicking on the "Hot Design header now does not directly open the HD overview page. You can compare with the [PR changes](https://unodocsprstaging.z13.web.core.windows.net/pr-20516/docs/articles/studio/studio-overview.html) Vs [Prod](https://platform.uno/docs/articles/studio/studio-overview.html)

Reply:
It seems to be a limitation. I'm not sure we can both link to a _sub TOC_ and have a link to a page on the same node.